.env.example

```diff
@@ -2,3 +2,12 @@ NODE_LOCAL_PORT=3000
 DB_URI=mongodb://127.0.0.1:27017/DB_NAME
```

Comment:
there's no variable for the TEST_DB_URI, please make sure that both the DB_URI and the TEST_DB_URI are hosted on Mongodb atlas